### PR TITLE
Optimize MapperManager

### PR DESCRIPTION
**Pull request by kang-hl**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Review by KomachiSion — CHANGES_REQUESTED** [review]
Is necessary to change ConcurrentHashMap in MapperManager?  I think there is no need, the MAPPER_SPI_MAP should only put in init step which is single thread. And even plugin implementation use join, the times is not many and frequent. So I think HashMap is enough and join with synchronized.

And other think is indent problem, please use nacos code style to reformat codes.

**Comment by kang-hl** [comment]
> Is necessary to change ConcurrentHashMap in MapperManager? I think there is no need, the MAPPER_SPI_MAP should only put in init step which is single thread. And even plugin implementation use join, the times is not many and frequent. So I think HashMap is enough and join with synchronized.
> 
> And other think is indent problem, please use nacos code style to reformat codes.

已修改

File: plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/MapperManager.java [hunk 1]
```diff
@@ -64,11 +64,10 @@ public static MapperManager instance(boolean isDataSourceLogEnable) {
     /**
      * The init method.
      */
-    public void loadInitial() {
+    public synchronized void loadInitial() {
```
Comment (KomachiSion) [inline plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/MapperManager.java:67]: 这方法有必要锁吗？ 只有初始化的时候单线程执行一次

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/MapperManager.java (hunk 1 above)
Comment (kang-hl) [reply plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/MapperManager.java:67]: > 这方法有必要锁吗？ 只有初始化的时候单线程执行一次

那就需要修改为private

**Review by KomachiSion — CHANGES_REQUESTED** [review]
能帮忙加下UT吗

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**